demo/local.satyh

```diff
@@ -1,16 +1,13 @@
-use package Stdlib
+use package open Stdlib.List
```

Comment:
The description `use package open Stdlib.List` looks like adding all identifiers in `Stdlib.List` (such as `map`, `fold-left`, etc.) to the current environment. I guess your intention is different; only `List` itself is wanted to be added to the current environment. It would thus be better to write `use package Stdlib.List` (i.e., to remove `open`).